grammars/php.cson

```diff
@@ -3724,3 +3734,32 @@
         ]
       }
     ]
+  
+  'ternary_shorthand':
+    'match': '\\s*(\\?:)\\s*'
+    'captures':
+      '1':
+        'name': 'keyword.operator.ternary.php'
+
+  'ternary_expression':
+    'begin': '(?<!\\?)\\s*(\\?)\\s*(?!:|\\?)'
+    'beginCaptures':
+      '1':
+        'name': 'keyword.operator.ternary.php'
+    'end': '\\s*(:)\\s*'
+    'endCaptures':
+      '1':
+        'name': 'keyword.operator.ternary.php'
+    'patterns': [
+      {
+        'include': '#language'
+      }
+      {
+        'include': '$self'
+      }
+    ]
+  'null_coalescing':
+    'match': '\\s*(\\?\\?)\\s*'
+    'captures':
+      '1':
+        'name': 'keyword.operator.null-coalescing.php'
```

Comment:
Add a newline after the last line

Reply:
Used to being spoiled by editorconfig. Good catch

Reply:
Done. ✅